docs/docsite/rst/conf.py

```diff
@@ -175,6 +175,10 @@
     'porting_guides/core_porting_guides',
 ] if tags.has('ansible') else '<UNKNOWN>'
 
+nitpick_ignore_regex = [
+    (r"py:.*", r"ansible\.module_utils\._internal\..*"),
```

Comment:
Not sure offhand what the consumer of this regex is paying attention to, but probably any  package matching `ansible\..*_internal\..*` should be ignored.

Reply:
Good point, fixed.